dev-jjjjjeong-bin/Invaders-SDP
# Integration of Effect Sound & BGM feature

## What
This PR adds the sound effect and BGM functionality for users. It includes the following changes:
- Added Ship & Enemy Ship shooting Sound Effect.
- Added UFO(special enemy) come-up & destroy Sound Effect.
- Added In-Game & Title Screen & Hall of Fame BGM.
- Added Ship & Enemy Ship destroy Sound Effect.
- Modify details(loudness, clarity) of ES, BGM. 

## Why
Sound functionality improves user experience while playing game.


## How to Test
1. Start the game
2. Check the sound correctly applied.


## Additional Information
1. We are going to apply sound of new functionality (PiercingBullet, Colored Enemy)
